hypershift-operator/controllers/nodepool/nodepool_controller.go

```diff
@@ -1449,3 +1451,13 @@ func validateInfraID(infraID string) error {
 	}
 	return nil
 }
+
+func setExpirationTimestampOnToken(ctx context.Context, c client.Client, tokenSecret *corev1.Secret) error {
+	// this should be a reasonable value to allow all in flight provisions to complete.
+	timeUntilExpiry := 2 * time.Hour
```

Comment:
bringing this to the top since the actual expiry time isn't mentioned in the PR description